### PR TITLE
Actualize data visualization palettes/themes according to the new generic color schemes

### PR DESCRIPTION
**Pull request by BingoRUS**

### CODE REVIEW

**Review by pavelgruba — CHANGES_REQUESTED** [review]
Please list newly added palettes as accepted values for CommonVizPalette option.

**Comment by BingoRUS** [comment]
@pavelgruba yeah thank you, forgot to update the accepted values.

File: js/viz/core/themes/generic.darkmoon.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    chart: {
+        commonPaneSettings: {
+            border: {
+                color: BORDER_COLOR
+            }
+        }
+    },
+    barGauge: {
+        backgroundColor: "#445066"
+    },
```
Comment (roman-simionov) [inline js/viz/core/themes/generic.darkmoon.js]: funnel: {
        item: {
            border: {
                color: "#465672"
            }
        }
    }

File: js/viz/core/themes/generic.darkviolet.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            }
+        }
+    },
+    chart: {
+        commonPaneSettings: {
+            border: {
+                color: BORDER_COLOR
+            }
+        }
+    }
```
Comment (roman-simionov) [inline js/viz/core/themes/generic.darkviolet.js]: `funnel: {
        item: {
            border: {
                color: "#465672"
            }
        }
    }`

**Review by roman-simionov — APPROVED** [review]

**Comment by BingoRUS** [comment]
@pavelgruba All additions are pushed, take a look, please.

**Review by pavelgruba — APPROVED** [review]

**Pull request merged**